doc/Gabc.tex

```diff
@@ -61,6 +62,30 @@ \subsection{Header}
   incipit, the first few words. In the case of the mass ordinary, the
   form as \texttt{Kyrie X Alme Pater} or \texttt{Sanctus XI} is
   recommended where appropriate. \textbf{This field is required.}
+\item[gabc-copyright] This license is the copyright notice (in english) of the gabc file, as chosen by the person named in the transcriber field. As well as the notice itself, it may include a brief description of the license, such as public domain, CC-by-sa; a separate text file will be necessary for the complete legal license. For the legal issues about gregorian chant scores, please see \url{http://gregorio-project.github.io/legalissues}. An example of this field would be:
+  \begin{lstlisting}[autogobble]
+    gabc-copyright: (C) Elie Roux, 2009, public domain;
+  \end{lstlisting}
+\item[score-copyright] This license is the copyright notice (in english) of the score itself from which the gabc was transcribed. Like the \texttt{gabc-copyright}, there may be a brief description of the license too. In unclear or complex cases it may be omitted; it is most suitable for use when the transcriber is the copyright holder and licensor of the score as well. One again, reading the page on legal issues (linked above) is recommended. An example of this field would be:
```

Comment:
why not, indeed

NB: CC0 is an edge case: for countries with copyright laws, it's public domain, for countries (like France) with author right laws, it's some equivalent of public domain, but not exactly public domain... so it's not really a copyright, but it's not really public domain...